source/_integrations/third_reality.markdown

```diff
@@ -0,0 +1,31 @@
+---
+title: Third Reality
+description: Connect and control your Third Reality Zigbee devices using the Zigbee integration
+ha_release: '2021.10'
```

Comment:
Which release supports this?

Reply:
The brand has been added in the current release.

Reply:
Current release would be 2022.10, not 2021.10, right?

Reply:
👀 
```suggestion
ha_release: '2022.10'
```